tests/python_functional/CMakeLists.txt

```diff
@@ -3,3 +3,6 @@ add_custom_target(pytest-self-check
 
 add_custom_target(pytest-check
    COMMAND ${PYTHON_EXECUTABLE} -m pytest ${PROJECT_SOURCE_DIR}/tests/python_functional/functional_tests --installdir=${CMAKE_INSTALL_PREFIX})
+
+add_custom_target(pytest-linters
+   COMMAND find ${PROJECT_SOURCE_DIR}/tests/python_functional/ -name "*.py" | xargs pre-commit run --show-diff-on-failure --config=${PROJECT_SOURCE_DIR}/tests/python_functional/.pre-commit-config.yaml --files)
```

Comment:
Just for optional, but piping find into xargs have some limitations/security risks. Some of them can be fixed with `find -print0 [...] | xargs -0 [...]`. It is also possible to work without xargs: using `find -exec {} \; [...]`

Reply:
+1 Thanks for the comment, agreed with you. I have also thinking about this possibility. I will choose one of them.

Reply:
fixed